tests/pytests/unit/modules/test_selinux.py

```diff
@@ -376,3 +378,36 @@ def test_selinux_config_disabled():
         for line in writes:
             if line.startswith("SELINUX="):
                 assert line == "SELINUX=disabled"
+
+
+@pytest.mark.parametrize(
+    "name,sel_type",
+    (
+        ("/srv/ssl/ldap/.*[.]key", "slapd_cert_t"),
+        ("/srv/ssl/ldap(/.*[.](pem|crt))?", "cert_t"),
+    ),
+)
+def test_selinux_add_policy_regex(name, sel_type):
+    """
+    Test adding policy with regex components parsing the stdout response of restorecon used in fcontext_policy_applied, new style.
+    """
+    mock_cmd_shell = MagicMock(return_value={"retcode": 0})
+    mock_cmd_run_all = MagicMock(return_value={"retcode": 0})
+
+    with patch.dict(selinux.__salt__, {"cmd.shell": mock_cmd_shell}), patch.dict(
+        selinux.__salt__, {"cmd.run_all": mock_cmd_run_all}
+    ):
+        selinux.fcontext_add_policy(name, sel_type=sel_type)
+        filespec = re.escape(name)
+        filespec_test = f"'{filespec}'"
+        expected_cmd_shell = f"semanage fcontext -l | egrep {filespec_test}"
```

Comment:
```suggestion
        expected_cmd_shell = f"semanage fcontext -l | egrep {filespec_test}"
```
Shouldn't it be expecting the single quotes?

Reply:
filespec_test has the quotes built into, see lines 401-402, do I see a ```expected_cmd_shell = f"semanage fcontext -l | egrep '{filespec_test}'"``` in my future :laughing: